boards/arm/sam4s_xplained/Makefile

```diff
@@ -0,0 +1,9 @@
+# Makefile - Atmel SAM4S Xplained Board
+#
+# Copyright (c) 2017 Justin Watson
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ccflags-y += -I$(srctree)/include/drivers
```

Comment:
Do you mean all of these are unnecessary? If I am to remove them all should I add the statement:
 
```
obj- += dummy.o
```